### PR TITLE
Update README.md - note for ARM

### PR DESCRIPTION
**Pull request by SwenSchaeferjohann**
make required use of devenv for tests on ARM explicit

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,15 +46,19 @@ Then, optionally, you can activate the development environment:
 . ./scripts/devenv.sh
 ```
 
-(The scripts mentioned later, like `./scripts/build.sh` or `./scripts/test.sh`
+> **Note:** If you're using an M1 or M2 Mac (ARM), you must always activate the development environment.
```
Comment (vadorovsky) [inline README.md:49]: Well, actually the whole paragraph is wrong. `build.sh` and `test.sh` never activate the devenv on their own, on any system. Sorry for misinformation.

I would rephrase it as:

> First, install the dependencies (they are going to be installed in `.local`
> directory inside your repository clone). If you want to install Redis (needed
> only for relayer), use `--enable-redis` option.
>
> ```shell
> ./scripts/install.sh
> ```
>
> Then you can activate the development environment:
>
> ```shell
> . ./scripts/devenv.sh
> ```
>
> Then follow the sections below, which describe the usage of `build.sh` and
> `test.sh` scripts.
>
> With activated development environment, you can manually run commands
> like `yarn`, `cargo`, `solana`, `solana-test-validator`. They are going to
> use the dependencies installed in `.local` directory, so even if you have
> different global installations, they are not going to interfere.

We could also eventually edit the "Manual setup" section and write there:

> ### Manual setup
>
> If you still want to setup dependencies manually, these are the requirements:
>
> * [Rust installed with Rustup](https://rustup.rs/), stable and nightly toolchains
> * [NodeJS](https://nodejs.org/) [(16.16 LTS)](https://nodejs.org/en/blog/release/v16.16.0)
> * [Anchor](https://www.anchor-lang.com/) [(0.28.0)](https://crates.io/crates/anchor-cli/0.28.0)
>
> Then follow the sections below, which describe the usage of `build.sh` and
> `test.sh` scripts.

**Review by vadorovsky — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (SwenSchaeferjohann) [reply README.md]: thx, updated based on your suggestion

**Review by vadorovsky — APPROVED** [review]
